static/app/controllers.js

```diff
@@ -11,23 +11,15 @@
             .then(renderStatus);
     });
 
-    app.controller('DashboardCtrl', function($scope, $cloudfoundry, $location) {
+    app.controller('MainCtrl', function($scope, $cloudfoundry, $location, $rootScope) {
```

Comment:
We should try to stay away from using rootscope
http://www.thinkingmedia.ca/2015/01/learn-how-to-use-scopes-properly-in-angularjs/#keep-the-rootscope-clean-of-variables
Instead we should use an injected service.